onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -1095,6 +1095,7 @@ ONNX_MS_OPERATOR_SET_SCHEMA(BeamSearch, 1,
                                 .Input(7, "vocab_mask", "Mask of vocabulary. Words that masked with 0 are not allowed to be generated, and 1 is allowed. Shape is (vacab_size)", "M", OpSchema::Optional)
                                 .Input(8, "prefix_vocab_mask", "Mask of vocabulary for first step. Words that masked with 0 are not allowed to be generated, and 1 is allowed. Shape is (batch_size, vocab_size)", "M", OpSchema::Optional)
                                 .Input(9, "attention_mask", "Custom attention mask. Shape is (batch_size, sequence_length)", "I", OpSchema::Optional)
+                                .Input(10, "timestamp_enable", "Enable timestamp processing. True means enabled. Shape is (1). Default  is ``False``", "tensor(bool)", OpSchema::Optional)
```

Comment:
Maybe we can export  time stamp processing to a subgraph, then call subgraph for logits processing. In this way, user can add its own logits processing

Reply:
Interesting idea. will think about it.

Reply:
Another solution is to change the inferface to more general, like logits_processor of tensor(int). So we can use value 1 to indicate time stamp. If other model has its own processing, it could use 2 etc for new processor.

Reply:
Done